server/views/pages/overview/partials/educationAndTrainingTab/_inPrisonQualifications.njk

```diff
@@ -1,52 +1,119 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds app-u-print-full-width">
 
-    <div class="govuk-summary-card">
-      <div class="govuk-summary-card__title-wrapper">
-        <h2 class="govuk-summary-card__title">In-prison qualifications and achievements</h2>
+    {% if featureToggles.newCourseAndQualificationHistoryEnabled %}
```

Comment:
Any thoughts on whether it might be better to do this as an entirely separate nunjucks template? This one looks to be getting quite large and difficult to read now?

Perhaps we could keep this one `_inPrisonQualifications.njk` exactly as is, then create a new one, perhaps called `_inPrisonQualificationsCompletedInLast12Months.njk` or similar? And then do the feature toggle switch in the template that includes this (so that it includes one or the other only)

What do you think?
It would also highlight the fact that in your current data model you have no way of knowing if there was a problem retrieving the data, so would not be able to display the fallback message . 🤔